### DISCUSSION TITLE
Actividad 1 - GRUPO 10

### DISCUSSION THREAD

**Discussion by sebasgonzales** [frlp-utn-ingsoft/catedra#100] [Show and tell]
https://github.com/MilagrosSoberon/vetsoft

Hicimos las actividades opciones en una rama "feature/add-foreign-keys", no está en develop ni en main porque lo único que no nos funcionó bien fue mostrar el historial clínico, después todo funcionaba correctamente.

En la rama "feature/repo-pet" un compañero hizo un missclick y la eliminó. Cuando dimos en reverse branch, quedó el commit de la branch eliminada, fue simplemente un missclick.

Todo lo que está en la rama main está en la rama develop, el merge entre ambas fue lo último que se hizo

Nosotros la clase pasada preguntamos si podíamos dar oral el siguiente lunes y nos habían dicho que sí.

**Comment by sebasgonzales** [comment]
Actualización: Ya funciona completamente la rama "feature/add-foreign-keys" donde están todas las tareas opcionales, pero cuando se crea un nuevo registro de historial médico, no se crea un nuevo objeto ya que utilizamos ManyToManyField,sino que se actualiza el de Pet, creemos que deberíamos crear un nuevo Model con ManyToOne que tenga los id de Pet, Vet y Medicine, y que ese objeto que cree la cantidad de veces que se haga un nuevo registro, ya que actualmente lo que hace es agregarlo como un array de los veterinarios y medicamentos seleccionados dentro de Pet, pero no sabíamos si haciendo eso rompíamos el modelo para los test, asi que por las dudas quedó así.
No está mergeado con develop ni con main ya que está terminado fuera del rango de entrega. :)

**Reply by sebasgonzales** [reply]
Actualización2: está todo en la rama main

**Comment by RodrigoJacznik** [comment]
Buenas!
La actividad está bien resuelta, queda aprobada.
Muy bueno y completo el trabajo  👏 👏

Saludos
